pkg/solana/chainwriter/transform_registry_test.go

```diff
@@ -76,16 +81,58 @@ func Test_CCIPExecuteArgsTransform(t *testing.T) {
 		args := struct {
 			ReportContext [2][32]uint8
 			Report        []uint8
+			Info          ccipocr3.ExecuteReportInfo
 		}{
 			ReportContext: [2][32]uint8{},
 			Report:        []uint8{},
+			Info: ccipocr3.ExecuteReportInfo{
```

Comment:
@amit-momin I was struggling to understand the purpose of this test, and I'm not sure if my additions interfere with what it's supposed to be testing...

Reply:
Ya I think this might conflict with what this was testing before. I was purposely trying to exclude a field to test the mapstructure decode we're using now. Before that would fail the `ReportPreTransform` type check we had. I think you can just remove `Report`. That should still test the same thing

Reply:
ok, done